### ISSUE TITLE
Create Webdriver has mutable default value

### ISSUE THREAD

**Issue by Snooz82** [robotframework/SeleniumLibrary#1817]
In src/SeleniumLibrary/keywords/browsermanagement.py

```
    @keyword
    def create_webdriver(
        self, driver_name: str, alias: Optional[str] = None, kwargs={}, **init_kwargs
    ) -> str:
```

the dictionary in kwargs should be replaced by `None`

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

**Comment by emanlove** [comment]
René, do you catch this with a linker/black?

**Comment by Snooz82** [comment]
@emanlove yes linter do catch this.

i can highly recommend `ruff` as a linter. It is amazingly fast and replaces isort and works well with black.

you can start with the config that Browser lib has in its pyproject.toml.

**Comment by emanlove** [comment]
Fixed in #1859
